.circleci/config.yml

```diff
@@ -7,11 +7,12 @@ env: &env
   environment:
     GRUNTWORK_INSTALLER_VERSION: v0.0.30
     TERRATEST_LOG_PARSER_VERSION: v0.30.4
-    MODULE_CI_VERSION: v0.29.1
-    TERRAFORM_VERSION: 0.13.5
+    MODULE_CI_VERSION: v0.31.0
```

Comment:
Update module CI to at least `v0.32.0`

Reply:
Nevermind, the v0.31.0 is the latest with the binaries fix.

Reply:
Changed to v0.33.2